corehq/apps/app_manager/models.py

```diff
@@ -4837,6 +4909,7 @@ def create_profile(self, is_odk=False, with_media=False, template='app_manager/p
         if toggles.CUSTOM_PROPERTIES.enabled(self.domain) and "custom_properties" in self__profile:
             app_profile['custom_properties'].update(self__profile['custom_properties'])
 
+        locale = self.langs[0] if not build_profile_id else self.build_profiles[build_profile_id].langs[0]
```

Comment:
@esoergel The ui wont let you save an app with zero languages so i think this is mainly an issue for tests were the app was instantiated without languages